firstvoices/backend/serializers/join_request_serializers.py

```diff
@@ -39,6 +43,34 @@ def validate(self, attrs):
             )
         return attrs
 
+    def create(self, validated_data):
+        logger = logging.getLogger(__name__)
+        created = super().create(validated_data)
+
+        site_language_admin_email_list = list(
```

Comment:
Optional: since there's a really well-defined set of logic here, I would lean towards putting it in a separate "notify_language_admins" method. Not a huge deal since not much else happens in this create method.